### ISSUE TITLE
[BUG] does not show the photo in full screen

### ISSUE THREAD

**Issue by OmlineEditor** [SimpleMobileTools/Simple-Gallery#2808]
the photos received from the scanner are not shown in full screen, I open them, but there is only a white or black rectum on the screen and there is nothing. copy some of the images to your phone and take a look. I have nothing showing

Simple Gallery Pro 6.26.5 (f-droid)
Androin 10
Phone: Redmi Note 7

[bug_foto.zip](https://github.com/SimpleMobileTools/Simple-Gallery/files/11029281/bug_foto.zip)

**Comment by tibbi** [comment]
and how do the real images look like? Not sure what is the issue here

**Comment by OmlineEditor** [comment]
> and how do the real images look like? Not sure what is the issue here

In reality, there are pieces of scanned image from my scanner (shoulder straps on the shoulder and flower), in the full -screen image only a white rectangle.

![3](https://user-images.githubusercontent.com/13348933/226659576-5022ce0b-54eb-4486-a0c5-6022cbc1394b.jpg)
![4](https://user-images.githubusercontent.com/13348933/226659582-c6a192bd-1394-4690-997a-40b294e67c64.jpg)

**Comment by tibbi** [comment]
do other galleries show them?

**Comment by OmlineEditor** [comment]
> do other galleries show them?

yes, of course. That's why I opened the ticket to tell you that you have a problem.
here is an example of the program QuickPic
![5](https://user-images.githubusercontent.com/13348933/226682424-d6f63c22-88df-4bbf-9441-bae9af58964b.jpg)

**Comment by tibbi** [comment]
it opens to me in Simple Gallery just fine though, could be some device specific glitch. Try restarting your device.

**Comment by OmlineEditor** [comment]
> it opens to me in Simple Gallery just fine though, could be some device specific glitch. Try restarting your device.

Reloading does not help. Where else could there be a problem?

**Comment by tibbi** [comment]
is it broken even if you disable "Deep zoomable images" toggle in the app settings?

**Comment by OmlineEditor** [comment]
> is it broken even if you disable "Deep zoomable images" toggle in the app settings?

 if you disable "Deep zoomable images"  - OK, works without problems.
it looks like the problem is only in this setting. can you fix the problem? maybe other users have problems too but can't tell you.

**Comment by tibbi** [comment]
well, hard to say what is the issue there, such gliches are usually difficult to solve or even reproduce.

**Comment by OmlineEditor** [comment]
if possible, study the problem and let the photos from the scanner show well.
